client/cli/lib/publish.ts

```diff
@@ -14,8 +14,9 @@ export default async (basePath: string, middleware?: Function, buildOptions?: Op
   const file = join(tmpdir(), filename);
 
   const { name, version, main, peerDependencies } = await build({ ...buildOptions, mode: Mode.production });
+  const distDir = join((buildOptions && buildOptions.dir) || '.', './dist');
```

Comment:
optionals chaining or default value?

Reply:
obviously a nitpick

Reply:
prettier is having issues with `buildOptions?.dir` and won't let me commit that